index.js

```diff
@@ -129,49 +118,38 @@ module.exports = {
       {
         arrays: 'always-multiline',
```

Comment:
I think this whole rule can be removed because it is the same as upstream:
https://github.com/airbnb/javascript/blob/711aeb650db14c3b8cc329511ac508ee05bc1af6/packages/eslint-config-airbnb-base/rules/style.js#L42